tests/core/test_accounts.py

```diff
@@ -90,21 +90,6 @@ def PRIVATE_KEY_ALT(request):
     return request.param
 
 
-@pytest.fixture
-def web3js_key():
```

Comment:
A long time ago this used to be used as a fixture, eg~ https://github.com/ethereum/eth-account/commit/d18f1454e0b9c48dd8c7be0086933e7327fb0e7b#diff-1de692d823115945c2232e1aa3066811R346

Then at some point, the method was converted from being used as a fixture, to being used as a simple method, but the change slipped in without removing the fixture decorator. Looks like here: https://github.com/ethereum/eth-account/commit/bc798a94883f3e6e3a49c176aa7c7887dd30b39e

Perhaps a simpler solution is to remove the `fixture` decorators here, and the next two methods. (Looks like this method could use a comment to clarify that this key is also the private key) We just want to test that account works when being passed a hex string or a native private key. :+1:

Reply:
Cool cool, I can add a follow-up PR for this. 👍

Reply:
Will actually update on #52 since that one is still open. Thanks!

Reply:
See: https://github.com/ethereum/eth-account/pull/52#issuecomment-453379730